src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -5670,6 +5710,17 @@ public sealed partial class SuppressMessageAttribute : System.Attribute
         public string? Scope { get { throw null; } set { } }
         public string? Target { get { throw null; } set { } }
     }
+    [System.AttributeUsageAttribute(System.AttributeTargets.All, Inherited=false, AllowMultiple=true)]
+    public sealed class UnconditionalSuppressMessageAttribute : System.Attribute
+    {
+        public UnconditionalSuppressMessageAttribute(string category, string checkId) { }
+        public string Category { get { throw null; } }
+        public string CheckId { get { throw null; } }
+        public string? Justification { get { throw null; } set { } }
+        public string? MessageId { get { throw null; } set { } }
+        public string? Scope { get { throw null; } set { } }
+        public string? Target { get { throw null; } set { } }
+    }
```

Comment:
Can we add basic tests as well for each of these attributes, e.g.
https://github.com/dotnet/runtime/blob/master/src/libraries/System.Diagnostics.Tools/tests/System/Diagnostics/CodeAnalysis/SuppressMessageAttributeTests.cs?

Reply:
Good point. I didn't know about these tests.

FYI - I'm going to put the new tests in `System.Runtime\tests`, since that is where these APIs are defined.